api/v1alpha4/virtualmachinegroup_types.go

```diff
@@ -0,0 +1,338 @@
+// © Broadcom. All Rights Reserved.
+// The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha4
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// VirtualMachineGroupMemberConditionGroupLinked indicates that the member
+	// exists and has its "Spec.GroupName" field set to the group's name.
+	VirtualMachineGroupMemberConditionGroupLinked = "GroupLinked"
+
+	// VirtualMachineGroupMemberConditionPowerStateSynced indicates that the
+	// member has been updated to match the group's power state.
+	VirtualMachineGroupMemberConditionPowerStateSynced = "PowerStateSynced"
+)
+
+const (
+	// LastUpdatedPowerStateTimeAnnotation is the annotation key for the last
+	// updated time of the power state.
+	LastUpdatedPowerStateTimeAnnotation = GroupName + "/last-updated-power-state-time"
+
+	// ScheduledPowerStateTimeAnnotation is the annotation key for the scheduled
+	// time when a power state change should be applied.
+	ScheduledPowerStateTimeAnnotation = GroupName + "/scheduled-power-state-time"
```

Comment:
What are these? Why do we have "scheduled" anything? We should not be scheduling anything.

Reply:
I _think_ I remember why you did this. Pinging you offline.

Reply:
Discussed offline. Going to remove these annotations from this /api package since they're not user facing.